composer.json

```diff
@@ -27,6 +28,7 @@
         "symfony/phpunit-bridge": "^2.7 || ^3.0"
     },
     "provide": {
+        "ext-mongo": "*",
```

Comment:
The "alcaeus/mongo-php-adapter" convert the mongo command for the old extension (ext-mongo) to the new one (ext-mongodb)